obspy/io/sh/core.py

```diff
@@ -404,7 +404,7 @@ def _read_q(filename, headonly=False, data_directory=None, byteorder='=',
             id = cid
             i += 1
         traces.setdefault(i, '')
-        traces[i] += line[3:].strip()
+        traces[i] += line[3:].rstrip('\n')
```

Comment:
I am not familiar with the SH file format at all so take this with a grain of salt: Why not just `rstrip()` to remove all trailing whitespaces? `\r\n` might be fairly common.